packages/console/src/pages/ConnectorDetails/index.tsx

```diff
@@ -108,11 +108,13 @@ const ConnectorDetails = () => {
       {error && <div>{`error occurred: ${error.body.message}`}</div>}
       {data && (
         <Card className={styles.header}>
-          {data.metadata.logo.startsWith('http') ? (
-            <img src={data.metadata.logo} className={styles.logo} />
-          ) : (
-            <ImagePlaceholder size={76} borderRadius={16} />
-          )}
+          <div className={styles.imagePlaceholder}>
+            {data.metadata.logo.startsWith('http') ? (
+              <img src={data.metadata.logo} className={styles.logo} />
+            ) : (
+              <ImagePlaceholder size={60} borderRadius={16} />
```

Comment:
Instead of manual input the ImagePlaceholder size & borderRadius, can we simply share the image styles between img element and the placeholder?. So they will always be the same.

Reply:
This is because the size of `img` inside `ImagePlaceholder` is affected by the outer `ImagePlaceholder`, so it's hard to have a common CSS to share the style here.
This is a temp solution, will be refactored once we have a real avatar.
Refer to this PR comment: https://github.com/logto-io/logto/pull/512#discussion_r844917270
<img width="812" alt="image" src="https://user-images.githubusercontent.com/10806653/164363667-6eeb575e-2804-436d-8097-780ce5e80d52.png">
Refer to the slack discussion: https://silverhand-io.slack.com/archives/C0243P3E4KS/p1649321315314789